examples/inverse/plot_use_sourcemorph.py

```diff
@@ -0,0 +1,250 @@
+"""
+================================
+Demonstrate usage of SourceMorph
+================================
+
+This example demonstrates how to morph an individual subject source estimate to
+a common reference space. It will be demonstrated using the SourceMorp class.
+The example uses parts of the MNE example
+:ref:`sphx_glr_auto_examples_inverse_plot_lcmv_beamformer_volume.py` and
+:ref:`sphx_glr_auto_examples_inverse_plot_mne_dspm_source_localization.py`.
+The respective result will be morphed based on an affine transformation and a
+nonlinear morph, estimated based on respective transformation from the
+subject's anatomical T1 (brain) to fsaverage T1 (brain) in VolSourceEstimate
+case and using an affine transform in the SourceEstimate or
+VectorSourceEstimate case. Afterwards the transformation will be applied to the
+beamformer result. The result will be a plot showing the morphed result
+overlaying the fsaverage T1. Uncomment at the respective location to plot the
+result of the surface morph.
+
+"""
+# Author: Tommy Clausner <tommy.clausner@gmail.com>
+#
+# License: BSD (3-clause)
+
+import matplotlib.pylab as plt
+import mne
+import nibabel  as nib
+import numpy as np
+from mne import SourceMorph
+from mne.beamformer import make_lcmv, apply_lcmv
+from mne.minimum_norm import make_inverse_operator, apply_inverse
+from nilearn.plotting import plot_anat
+
+print(__doc__)
+
+
+###############################################################################
+# from :ref:`sphx_glr_auto_examples_inverse_plot_mne_dspm_source_localization.py`.
+# Note, that in order to save the data fname needs to be set
+def compute_lcmv_example_data_surf(data_path, fname=None):
```

Comment:
you call this function only once and this is not meant to become an mne core function. So don't use a function here.

Reply:
this still needs to be fixed